functests/5_latency_testing/latency_testing.go

```diff
@@ -122,8 +122,11 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 				testlog.Error(matchErr.Error())
 			}
 			if ok {
-				commandRegex := fmt.Sprintf("%s command with arguments .*--duration %s", test.toolToTest, test.testRuntime)
-				Expect(string(output)).To(MatchRegexp(commandRegex), "The output of the executed tool is not as expected")
+				//verify the command is executed with the expected args
+				args := getToolArgsRegex(test)
```

Comment:
I don't really like this for a couple reasons:
1. this is actually hiding more (should be less) the purpose of the test. Test code should be way of factoring and adding helpers, because the test code has a different purpose than production code (https://about.sourcegraph.com/go/advanced-testing-in-go/#Repeat-yourself)
2. this doesn't seem to return regexes - rather plain strings and the purpose of the code is not really clear here

Reply:
this link should work: https://about.sourcegraph.com/go/advanced-testing-in-go/#Repeat-yourself